src/components/list/Form.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const Form = () => (
+  <div className="form">
+    <h3>ADD NEW BOOK</h3>
+    <form>
+      <input type="text" />
+      <select>
+        <option value="Categories">Categories</option>
+        <option value="Books">Books</option>
+      </select>
```

Comment:
- [ ] Kindly make sure your book component renders a list of books with a ` title` and `author` on the webpage 🙏 . This can be hardcoded as no other functionality is needed right now.
- [ ] Great job 👏 . Please, create a prop for the book component and it should render the title and author of the book as a prop. 🙏 . You can read more about props in this [article](https://reactjs.org/docs/components-and-props.html) and [video](https://youtu.be/PHaECbrKgs0)